packages/strapi/lib/services/entity-service.js

```diff
@@ -128,13 +128,13 @@ const createDefaultImplementation = ({ db, eventHub, entityValidator }) => ({
    * @param {object} ctx.model - Model that is being used
    */
   async create(opts, { model }) {
-    const { data, files } = await this.wrapOptions(opts, { model });
+    const { data, files, params } = await this.wrapOptions(opts, { model });
 
     const modelDef = db.getModel(model);
 
     if (modelDef.kind === 'singleType') {
```

Comment:
I actually think we can remove this. it won't really make much sense here with the possibility to actually create more versions of one single type. We can move this logic into the coreAPI single type service to avoid errors on API calls but for the CM it isn't necessary as we already do that checking logic too

Reply:
Ok, I deleted it then